package.json

```diff
@@ -21,16 +21,16 @@
     "build": "babel src --out-dir build",
     "clean": "rimraf build",
     "test:lint": "standard --verbose | snazzy",
-    "test:unit": "cross-env NODE_ENV=test mocha test/**/*.test.js",
+    "test:unit": "cross-env NODE_ENV=test mocha --require test/enzyme-preamble.js test/**/*.test.js",
     "test:watch": "npm run test:unit -- --watch",
     "test": "npm run test:lint && npm run test:unit"
   },
   "peerDependencies": {
     "chai": "^3.0.0 || ^4.0.0",
     "cheerio": "0.19.x || 0.20.x || 0.22.x || 1.0.0-rc.1",
-    "enzyme": "1.x || ^2.3.0",
-    "react": "^0.14.0 || ^15.0.0-0",
-    "react-dom": "^0.14.0 || ^15.0.0-0"
+    "enzyme": "^3.0.0 || ^3.0.0-rc || ^3.0.0-beta || ^3.0.0-alpha",
```

Comment:
this library was previously using an internal file of enzyme to do `findDOMNode` which we should probably avoid as it could have broken at any time from a change that we would not have considered "breaking". I can't think of a quick way to make `getDOMNode()` supported across all of the react versions without doing something somewhat hacky. Maintaining 2.x support seems pretty reasonable to me? If people are upgrading chai-enzyme, it seems like they'd be upgrading enzyme as well.